Jenkinsfile

```diff
@@ -0,0 +1,8 @@
+overrides = [
+  clone: [
+    gitServer: 'git@github.com',
+    repoBranch: 'origin/master',
+  ],
+  platforms: ['trusty', 'xenial', 'bionic']
+]
+buildDebianPackage(overrides)
```

Comment:
Just confirming, we need both the Jenkinsfile and jenkins.yaml? I thought only one was used.

Reply:
The jenkins.yaml isn't going to be honoured by automated scripts since this is a public repo. I added it for consistency, but it's not used.

Reply:
Should we delete it if it's unused? I think it will cause confusion by being here.